integrationtests/spring-boot-it/src/test/java/org/infinispan/integrationtests/spring/boot/session/remote/RemoteConfiguration.java

```diff
@@ -18,13 +20,14 @@
 @Import(WebConfig.class)
 public class RemoteConfiguration {
 
-   public static final int SERVER_PORT = SocketUtils.findAvailableTcpPort();
+   static final int SERVER_PORT = SocketUtils.findAvailableTcpPort();
 
    @Bean
    public SpringRemoteCacheManagerFactoryBean springCacheManager() {
       SpringRemoteCacheManagerFactoryBean factoryBean = new SpringRemoteCacheManagerFactoryBean();
       factoryBean.setServerList(Arrays.asList(new InetSocketAddress("localhost", SERVER_PORT)));
+      factoryBean.setMarshaller(JavaSerializationMarshaller.class.getName());
```

Comment:
this shouldn't be necessary at this point, am I wrong ?

Reply:
Good spot, removed!